### DISCUSSION TITLE
Forms

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#425] [Ideas]
I know it's still too early to talk about forms, just wanted to share what I just saw in GitHub Projects.

What I'm certain we'll need is the configuration of fields to show in a form and this seems to be a good example of that.

https://github.com/user-attachments/assets/5ad224e0-0aa0-48d5-883d-75519688072a

In the Project settings, there are a few field types to choose from.
Then they become available in the form. But here I don't think GitHub has an option to enforce filling certain fields.

**Comment by DenisMGojak** [comment]
I belive it's the right path. We can have an "Form Engine" capabale of tailoring any kind of forms. The forms creator (Forms admin) can than "publish" them and give access rights. Looking forward for this!
